### PR TITLE
Update ppp

### PR DESCRIPTION
**Pull request by joelngyx**
Updated and completed the PPP.

### CODE REVIEW

**Review by anqi20 — COMMENTED** [review]
Joel, I think your formatting is a little different from the rest of us. Do you want to look at ours first?

File: docs/team/joelngyx.md [hunk 1]
```diff
@@ -19,37 +17,37 @@ E-Duke-8 (pronounced "Educate") helps CS2113/T students learn and understand sof
...
-### Contributions to team-based tasks:
-
-### Review contributions:
-
-- PRs reviewed:
-  - 
-  -
+  - Justification: Necessary to process and carry out the NoteCommand. 
+  
+  Code contributed: [RepoSense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=joelngyx&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=false)
```
Comment (kstonekuan) [inline docs/team/joelngyx.md:26]: Good formatting

File: docs/team/joelngyx.md [hunk 2]
```diff
@@ -19,37 +17,37 @@ E-Duke-8 (pronounced "Educate") helps CS2113/T students learn and understand sof
...
-  - Added UML and sequence diagrams relevant to the Topic, TopicList, Note and NoteList components [Link](TBC).
-
-### Contributions to team-based tasks:
-
-### Review contributions:
-
-- PRs reviewed:
-  - 
-  -
+  - Justification: Necessary to process and carry out the NoteCommand.
```
Comment (kstonekuan) [inline docs/team/joelngyx.md:24]: Would be good to add some highlights to know which areas were challenging

**Review by kstonekuan — APPROVED** [review]
LGTM

**Pull request merged**